rb/lib/selenium/webdriver/common/driver.rb

```diff
@@ -314,13 +314,22 @@ def create_bridge(caps:, url:, http_client: nil)
         end
       end
 
-      def process_options(options, capabilities)
+      def process_options(options, capabilities, klass = nil)
```

Comment:
I suppose you don't have explicitly pass this and you can try to deprecate based on `self.class` being a kind of `Chrome::Driver`, `Firefox::Driver`, etc.

Reply:
Hmm, ok, yeah I'm using it to differentiate whether it is a local or remote driver, and I can do that more easily since we aren't subclassed any more.